test/controllers/auth/GmpGovernmentGatewaySpec.scala

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright 2019 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package controllers.auth
-
-import org.scalatestplus.play.{OneServerPerSuite, PlaySpec}
-
-class GmpGovernmentGatewaySpec extends PlaySpec with OneServerPerSuite {
-  
-  "GmpGovernmentGateway" must {
-    
-    "have values overridden" in {
-      GmpGovernmentGateway.loginURL must be(ExternalUrls.signIn)
```

Comment:
We need a test for urls that it sends you to login

Reply:
@Froom2 All other issues have been addressed so if you can take a look at this in the morning it's then ready to go i think.